structural-annotation.wdl

```diff
@@ -1,72 +1,69 @@
+version 1.0
+
 import "trnascan.wdl" as trnascan
 import "rfam.wdl" as rfam
 import "crt.wdl" as crt
 import "cds_prediction.wdl" as cds_prediction
 
 workflow s_annotate {
-  String  cmzscore
-  File    imgap_input_fasta
-  String  imgap_project_id
-  String  imgap_project_type
-  Int     additional_threads
-  Int? imgap_structural_annotation_translation_table
-  Boolean pre_qc_execute=false
-  Boolean trnascan_se_execute=true
-  Boolean rfam_execute=true
-  Boolean crt_execute=true
-  Boolean cds_prediction_execute=true
-  Boolean prodigal_execute=true
-  Boolean genemark_execute=true
-  Boolean gff_and_fasta_stats_execute=true
-  String  database_location
-  String  container
-  String? gm_license
-  String pre_qc_container = "microbiomedata/mg-annotation:0.1.2"
-
-
-
-  if(pre_qc_execute) {
-    call pre_qc {
-      input:
-        project_type = imgap_project_type,
-        input_fasta = imgap_input_fasta,
-        project_id = imgap_project_id,
-        container=container
+    input {
+      String  cmzscore
+      File    imgap_input_fasta
+      String  imgap_project_id
+      String  imgap_project_type
+      Int     additional_threads
+      Int? imgap_structural_annotation_translation_table
+      Boolean pre_qc_execute=false
+      Boolean trnascan_se_execute=true
+      Boolean rfam_execute=true
+      Boolean crt_execute=true
+      Boolean cds_prediction_execute=true
+      Boolean prodigal_execute=true
+      Boolean genemark_execute=true
+      Boolean gff_and_fasta_stats_execute=true
+      String  database_location
+      String  container
+      String gm_license="/refdata/licenses/.gmhmmp2_key"
     }
-  }
 
-  if(trnascan_se_execute) {
```

Comment:
be consistent, if you are removing the boolean check also remove the boolean variable